cfme/configure/configuration/server_settings.py

```diff
@@ -18,8 +18,8 @@
 class ServerInformationView(ServerView):
     """ Class represents full Server tab view"""
     title = Text("//div[@id='settings_server']/h3[1]")
-    save_button = Button('Save')
-    reset_button = Button('Reset')
+    save = Button('Save')
```

Comment:
Should likely use the `title='Save Changes'` button here as well because of the buttons_on/buttons_off divs.

calls to disabled/active won't work because of the DOM difference, but those are the buttons that are actually in buttons_on div, and only match there instead of matching the disabled and enabled buttons